DaysOfCode/classVsInstance.js

```diff
@@ -1,60 +1,71 @@
+/* eslint-disable no-unused-vars */
 /* eslint-disable prefer-const */
 /* eslint-disable radix */
-
 let inputStdin = '';
 let inputStdinArray = '';
 let inputCurrentline = 0;
 
 process.stdin.resume();
 process.stdin.setEncoding('ascii');
 process.stdin.on('data', (data) => {
-    inputStdin += data;
+  inputStdin += data;
 });
 
 process.stdin.on('end', () => {
-    inputStdinArray = inputStdin.split('\n');
-    main();
+  inputStdinArray = inputStdin.split('\n');
+  main();
 });
 
 function readLine() {
-    return inputStdinArray[inputCurrentline += 1];
+  return inputStdinArray[inputCurrentline += 1];
 }
 
 function Person(initialAge) {
-    let age = 0;
-    // Add some more code to run some checks on initialAge
-    if (initialAge < 0) {
-        console.log('Age is not valid, setting age to 0.');
-        age = 0;
-    } else {
-        age = initialAge;
+  const age = 0;
+
+  function setAge(newAge) {
+    this.age = newAge;
+  }
+  function getAge() {
+    return this.age;
```

Comment:
not a fan either, you have a raw function here, if you want to act like a class, use es6 classes..